packages/core/src/App/Containers/AccountSignupModal/account-signup-modal.jsx

```diff
@@ -45,25 +45,31 @@ const validateSignup = (values, residence_list) => {
     return errors;
 };
 
-class AccountSignup extends React.Component {
-    static contextType = PlatformContext;
-    state = {
-        has_valid_residence: false,
-        pw_input: '',
-    };
+const AccountSignup = ({
+    enableApp,
+    isModalVisible,
+    is_account_signup_modal_visible,
+    is_eu,
```

Comment:
`is_eu` is provided from [AccountSignupModal as props](https://github.com/binary-com/deriv-app/pull/2241/files#diff-5b31e01760d5b651118144eb351cdb43b32cb0c870ef5fe3c141b194f3a7cab7R227-R234) but it is not used inside of AccountSignup Component. do we need it?

Reply:
we can remove `is_eu` if the prop is not used